frontend/src/components/course/CourseView.js

```diff
@@ -0,0 +1,88 @@
+import React, { useState } from 'react'
+
+import { useQuery, useMutation } from 'react-apollo-hooks'
+import { LINK_PREREQUISITE, DELETE_LINK, CREATE_CONCEPT } from '../../services/ConceptService'
+import {
+  ALL_COURSES,
+  FETCH_COURSE,
+  ADD_COURSE_AS_PREREQUISITE,
+  COURSE_PREREQUISITE_COURSES
+} from '../../services/CourseService'
+
+import CourseContainer from './CourseContainer'
+import CourseTray from './CourseTray'
+import ActiveCourse from './ActiveCourse'
+
+const CourseView = ({ course_id }) => {
+  const [activeConceptId, setActiveConceptId] = useState('')
+
+  const courses = useQuery(ALL_COURSES)
+
+  const course = useQuery(FETCH_COURSE, {
+    variables: { id: course_id }
+  })
+
+  const prerequisites = useQuery(COURSE_PREREQUISITE_COURSES, {
+    variables: { id: course_id }
+  })
+
+  const linkPrerequisite = useMutation(LINK_PREREQUISITE, {
+    refetchQueries: [{ query: ALL_COURSES }]
+  })
+
+  const deleteLink = useMutation(DELETE_LINK, {
+    refetchQueries: [{ query: ALL_COURSES }]
+  })
+
+  const addCourseAsPrerequisite = useMutation(ADD_COURSE_AS_PREREQUISITE, {
+    refetchQueries: [{
+      query: COURSE_PREREQUISITE_COURSES,
+      variables: { id: course_id }
+    }]
+  })
+
+  const createConcept = useMutation(CREATE_CONCEPT, {
+    refetchQueries: [{
+      query: ALL_COURSES
+    }]
```

Comment:
FETCH_COURSE was creating unexpected behavior, and is not refetched after creating concept for course. Need to investigate further.